topcoffea/cfg/mc_background_samples.cfg

```diff
@@ -3,17 +3,17 @@
 # Central UL16APV background samples
 #root://ndcms.crc.nd.edu/
 root://deepthought.crc.nd.edu/
-../../topcoffea/json/background_samples/central_UL/UL16APV_ZGToLLG.json
-#../../topcoffea/json/background_samples/central_UL/UL16APV_TTTo2L2Nu.json
-#../../topcoffea/json/background_samples/central_UL/UL16APV_TTToSemiLeptonic.json
+#../../topcoffea/json/background_samples/central_UL/UL16APV_ZGToLLG.json
```

Comment:
Are the `ZG` conversions just for the CR plots, or do we need to uncomment them.

Reply:
The ZG conversions eventually may be used for the SR as well, but first we'll probably want to check how large the effect is in of the SR channels. So, for this run, I was not going to include them, so left it commented for now.

Reply:
Sounds good.